src/main/kotlin/biokotlin/seqIO/FastaIO.kt

```diff
@@ -177,3 +178,27 @@ fun writeFasta(input: SequenceIterator, filename: String) {
     }
 
 }
+
+fun writeFasta(input: Collection<SeqRecord>, filename: String) {
+    val extension = File(filename).extension
+    val newFilename = if (extension.equals("fa", false) || extension.equals("fasta", false)) {
+        filename
+    } else {
+        "$filename.fa"
+    }
+
+    println("FastaIO: writeFasta: writing file $newFilename")
+
+    bufferedWriter(newFilename).use { outputFile ->
+        for (record in input) {
+            val seq = when (record) {
+                is NucSeqRecord -> record.sequence.toString()
+                is ProteinSeqRecord -> record.sequence.toString()
+                else -> throw IllegalStateException("writeFasta trying to output something other than Nuc or Protein Seq.")
+            }
+            outputFile.write(">${record.id}\n")
+            outputFile.write(seq)
```

Comment:
Do we want to limit the length of the lines. Maybe like this?
seq
            .chunked(60)
            .forEach { chunk ->
                writer.write(chunk)
                writer.newLine()
            }

Reply:
Thank you for your suggestion, but could you clarify why limiting the length of the lines is necessary?

Reply:
My suggested code breaks up the sequence into 60 character chunks instead of having one long line

Reply:
That makes sense, I updated the program so that it breaks up the sequence into 60 character chunks